client/src/components/Browse/Sources/BrowseSources.tsx

```diff
@@ -22,107 +31,128 @@ export const BrowseSources = () => {
-                <div className="source-item-in-group">gene claims in group: {src.drugClaimsInGroupsCount}</div>
-                <div className="source-item-links">
-                  <div className="source-item-license"><a>License</a></div>
-                  <div className="source-item-citation"><a>Full Citation</a></div>
-                </div>
-              </div>
-            )
-          })
-        }
-      </div>
-
-      <div className="source-type-header"><h2><b>Potentially Druggable</b></h2></div>
-      <div className="sources-grid">
-        {
-          interactionSources?.map((src: any) => {
-            return (
-              <div className="gene-source-item">
-                <div className="source-item-name">{src.sourceDbName}</div>
-                <div className="source-item-count">gene claims count: {src.drugClaimsCount}</div>
-                <div className="source-item-in-group">gene claims in group: {src.drugClaimsInGroupsCount}</div>
-                <div className="source-item-count">gene claims count: {src.geneClaimsCount}</div>
-                <div className="source-item-in-group">gene claims in group: {src.geneClaimsInGroupsCount}</div>
-                <div className="source-item-links">
-                  <div className="source-item-license"><a>License</a></div>
-                  <div className="source-item-citation"><a>Full Citation</a></div>
-                </div>
-              </div>
-            )
-          })
-        }
-      </div>
-
-      <div className="source-item">
-        <div className="source-title">
-        </div>
-      </div>
-    </div>
+    <Box className="sources-page-container">
+      <Box display="flex"><h1 style={{marginRight: "10px"}}><b>Sources</b></h1> <Box my="auto">details about data sources in DGIdb</Box></Box>
+      <Box mb="20px">
+        <ButtonGroup color="primary" onClick={handleButtonClick} className="filter-buttons">
+          <Button variant={filter === "All" ? "outlined" : "contained"} 
+            value="All">
+              All
+            </Button>
+          <Button
+            variant={filter === "Gene" ? "outlined" : "contained"} 
+            value="Gene">
+            Gene
+          </Button>
+          <Button
+            variant={filter === "Drug" ? "outlined" : "contained"} 
+            value="Drug">
+            Drug
+          </Button>
+          <Button
+            variant={filter === "Interaction" ? "outlined" : "contained"} 
+            value="Interaction">
+            Interaction
+          </Button>
+          <Button
+            variant={filter === "Potentially Druggable" ? "outlined" : "contained"} 
+            value="Potentially Druggable">
+            Potentially Druggable
+          </Button>
+        </ButtonGroup>
+      </Box>
+      {
+        sectionsMap.map((section: any) => {
+          return section.value === filter || filter === "All" ? (
+            <>
```

Comment:
I don't think it matters but I'm a stickler for warnings - this `map` call, and the one on line 147, both throw the react `each child needs a key prop` warning. Should be a pretty easy fix